### DISCUSSION TITLE
Inline completions are not triggered when i am on the same line

### DISCUSSION THREAD

**Discussion by dreambit** [community/community#70007] [Copilot Conversations]
### Select Topic Area

Bug

### Body

Inline completions are not triggered when I am on the same line. I have to put a line break to make it work.

https://github.com/community/community/assets/1479661/72c252b0-5c44-4f0f-80ed-0dc1ee4bcc92

**Comment by Akash1134** [comment]
Hi @dreambit ,

We've seen this issue reported in the past. Would you mind checking this [discussion](https://github.com/orgs/community/discussions/48470) and let us know if that helps?

Thanks!

**Reply by dreambit** [reply]
Hi, it is for vscode.
inline completions work, but only when a new line is added, could you please try with my example?

**Reply by dreambit** [reply]
Any updates?

**Comment by dreambit** [comment]
Any updates?

**Comment by mateuszmazurek** [comment]
This behavior drives me crazy. I used Codeium (free) for code completions and I feel like it works much better than paid Github Copilot 😐

**Reply by dreambit** [reply]
Yes!!! exactly, i use Codeium as a workaround for not working inline completions and it is providing better suggestions, especially in chat
